crates/matrix-sdk-base/src/client.rs

```diff
@@ -288,15 +286,21 @@ impl BaseClient {
         &self.event_cache_store
     }
 
-    /// Is the client logged in.
-    pub fn logged_in(&self) -> bool {
+    /// Check whether the client has been activated.
+    ///
+    /// See [`BaseClient::activate`] to know what it means.
+    pub fn is_activated(&self) -> bool {
```

Comment:
`has_been_activated` or `is_active` sounds like the more correct grammar here, like you used in the description of this method :sweat_smile:.

Reply:
🤦  done!